nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json

```diff
@@ -0,0 +1,115 @@
+{
+    "mappings": {
+        "dynamic": "strict",
+        "properties": {
+            "date_end_project": {
+                "type": "date"
+            },
+            "date_start_project": {
+                "type": "date"
+            },
+            "json_funding_project": {
+                "properties": {
+                    "amount": {
+                        "type": "integer"
+                    },
+                    "category": {
+                        "type": "keyword"
+                    },
+                    "currencyCode": {
+                        "type": "keyword"
+                    },
+                    "end": {
+                        "type": "date"
+                    },
+                    "start": {
+                        "type": "date"
+                    }
+                },
+                "type": "nested"
+            },
+            "json_outcomes_project": {
+                "dynamic": true,
+                "type": "nested"
+            },
+            "name_leadOrgDepartment_project": {
+                "fields": {
+                    "keyword": {
+                        "type": "keyword"
+                    }
+                },
+                "type": "text"
+            },
+            "name_of_funder": {
+                "fields": {
+                    "keyword": {
+                        "type": "keyword"
+                    }
+                },
+                "type": "text"
+            },
+            "status_of_project": {
+                "type": "keyword"
+            },
+            "terms_continent_project": {
+                "type": "keyword"
+            },
+            "terms_countries_project": {
+                "type": "keyword"
+            },
+            "terms_instituteIds_project": {
+                "type": "keyword"
+            },
+            "terms_institutes_project": {
+                "type": "keyword"
+            },
+            "terms_iso2_project": {
+                "type": "keyword"
+            },
```

Comment:
What about `terms_countryId_project` with the assumption that we always use ISO alpha 2.

Can we add some custom doc string in ES mappings? Like:

```json
"terms_iso2_project": {
  "type": "keyword"
  "__doc": "country ISO alpha 2 codes"
},
```

Reply:
regarding `countryId` - for consistency i will stick with `iso2`, and then this issue #285 will consider replacing `iso2` --> `countryId` so that all indexes can be consistent.

regarding the docstring - it doesn't seem so - *but* there's nothing stopping us adding custom docstrings and then stripping them out again so I added an issue #284

Reply:
awesome, thanks a lot